packages/react-core/src/components/FileUpload/FileUploadField.tsx

```diff
@@ -84,6 +86,7 @@ export interface FileUploadFieldProps extends Omit<React.HTMLProps<HTMLDivElemen
 
 export const FileUploadField: React.FunctionComponent<FileUploadFieldProps> = ({
   id,
+  name,
```

Comment:
should there be a default value? currently breaking change as we are removing the `{id}-filename` default

Reply:
I agree.  Can we make it a default value.